packages/app/src/systems/Assets/containers/AssetsDialog.tsx

```diff
@@ -117,6 +130,18 @@ export function AssetsDialog() {
                       : undefined
                   }
                   isFaucetLoading={isFaucetable && isLoadingFaucet}
+                  onAddToWallet={
```

Comment:
imo we should hide the add to wallet button if neither of the wallets are connected

Reply:
agree